intellij-plugin-structure/structure-ktor/src/main/kotlin/com/jetbrains/plugin/structure/ktor/KtorFeaturePluginManager.kt

```diff
@@ -108,6 +108,7 @@ class KtorFeaturePluginManager private constructor(private val extractDirectory:
               options = doc.options!!
             )
           },
+          requiredFeatures = this.requiredFeatures,
```

Comment:
Can we also add validation that this feature id exists in MP?

Reply:
yes